hmpps-prisoner-search-indexer/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/services/PrisonerSynchroniserService.kt

```diff
@@ -126,7 +126,7 @@ class PrisonerSynchroniserService(
       prisonerRepository.save(it, SyncIndex.RED)
     }
 
-  internal fun compareAndMaybeIndex(ob: OffenderBooking, indices: List<SyncIndex>) {
+  internal fun compareAndMaybeIndex(ob: OffenderBooking, indices: List<SyncIndex>, label: String) {
     val incentiveLevel = getIncentive(ob)
```

Comment:
won't you get the incentive and restricted patient twice - one for the green/blue and once for red?  That will nearly double the amount of requests that the refresh makes